src/features/dashboard/ui/ResponsesList.tsx

```
@@ -1,121 +1,95 @@
-        return (
-          <>
-            {renderSection('name', 'name', true)}
-            {renderSection('phone', 'phone', true)}
-            {renderSection('email', 'email', true)}
-            {renderSection('grade', 'grade', true)}
-          </>
+        const filteredResponses = ticketOptionResponses.filter(
+          (option) => option.optionType !== 'TEXT'
         );
-      case 'query':
+        console.log(filteredResponses)
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)